### PR TITLE
COMMERCE-9538 | master

### PR DESCRIPTION
**Pull request by Kvale1733**
https://issues.liferay.com/browse/COMMERCE-9538

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
ci:test:commerce-functional

**Comment by Kvale1733** [comment]
ci:test:sf

**Comment by Kvale1733** [comment]
ci:test:commerce-functional

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 1]
```diff
@@ -1116,6 +1116,73 @@ definition {
...
+			CommerceNavigator.gotoEntry(entryName = "Minium Portal");
+		}
+
+		task ("Then the buyer is unable to change the channel currency code") {
+			AssertElementNotPresent(
+				key_title = "Currency",
+				locator1 = "CommerceEntry#CATALOGS_FORM");
+		}
+	}
+
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:1190]: @Kvale1733 could you please move this test inside `CPCommerceChannels.testcase`? Thanks!

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase [hunk 2]
```diff
@@ -1061,13 +1120,79 @@ definition {
 			locator1 = "CommerceFrontStore#FS_PRODUCT_SKU");
 	}
 
+	@description = "Verify buyers cannot change the currency of a channel."
+	@priority = "3"
+	test BuyersCannotEditChannelCurrency {
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductCard.testcase:1125]: @Kvale1733 in accordance with @victoroscuro, please change the name of this test to this:
```suggestion
	@description = "Verify users without edit permission cannot change the currency of a channel."
	@priority = "3"
	test CanNotEditChannelCurrencyAsUserWithoutEditPermission {
```

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by Kvale1733** [comment]
ci:test:sf

**Review by lorenzolapescara — APPROVED** [review]

**Pull request closed without merging**